README.md

```diff
@@ -48,3 +48,54 @@ default VPC in every region for that account.
 No outputs.
 
 <!-- END TFDOCS -->
+
+## CLI Option
+
+Steps to run via the CLI
+
+1. Install and configure aws cli.
+2. Set AWS_DEFAULT_PROFILE and AWS_DEFAULT_REGION (account and region that can assume the role and run commands from)
```

Comment:
Suggest just using `AWS_PROFILE`, as it always overrides the "default" profile. I.e. if they had `AWS_PROFILE` set, then `AWS_DEFAULT_PROFILE` would not work.

```suggestion
2. Set AWS_PROFILE and AWS_DEFAULT_REGION (account and region that can assume the role and run commands from)
```